test/helpers/lightning.ts

```diff
@@ -0,0 +1,260 @@
+import { once } from "events"
+import { sleep } from "src/utils"
+import { baseLogger } from "src/logger"
+import { offchainLnds, onchainLnds, onChannelUpdated, updateEscrows } from "src/lndUtils"
+import {
+  AuthenticatedLnd,
+  authenticatedLndGrpc,
+  closeChannel,
+  createChainAddress,
+  deletePayments,
+  getChainBalance,
+  getChannelBalance,
+  getChannels,
+  getPendingChannels,
+  getWalletInfo,
+  openChannel,
+  sendToChainAddress,
+  subscribeToChannels,
+  subscribeToGraph,
+  updateRoutingFees,
+} from "lightning"
+
+import { bitcoindClient, RANDOM_ADDRESS } from "./bitcoinCore"
+
+export * from "lightning"
```

Comment:
what is the goal of this?

Reply:
I tried to move moved all external references in tests to helpers, this is the first step to a next stage where we can mock classes, functions, etc to add more tests without runtime environment dependency

Reply:
ok